xml/Microsoft.Extensions.DependencyInjection/HttpClientBuilderExtensions.xml

```diff
@@ -1,1033 +1,1060 @@
+      <MemberSignature Language="DocId" Value="M:Microsoft.Extensions.DependencyInjection.HttpClientBuilderExtensions.AddLogger``1(Microsoft.Extensions.DependencyInjection.IHttpClientBuilder,System.Boolean)" />
+      <MemberSignature Language="VB.NET" Value="&lt;Extension()&gt;&#xA;Public Function AddLogger(Of TLogger As IHttpClientLogger) (builder As IHttpClientBuilder, Optional wrapHandlersPipeline As Boolean = false) As IHttpClientBuilder" />
+      <MemberSignature Language="F#" Value="static member AddLogger : Microsoft.Extensions.DependencyInjection.IHttpClientBuilder * bool -&gt; Microsoft.Extensions.DependencyInjection.IHttpClientBuilder (requires 'Logger :&gt; Microsoft.Extensions.Http.Logging.IHttpClientLogger)" Usage="Microsoft.Extensions.DependencyInjection.HttpClientBuilderExtensions.AddLogger (builder, wrapHandlersPipeline)" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Microsoft.Extensions.Http</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>Microsoft.Extensions.DependencyInjection.IHttpClientBuilder</ReturnType>
+      </ReturnValue>
+      <TypeParameters>
+        <TypeParameter Name="TLogger">
+          <Attributes>
+            <Attribute>
+              <AttributeName Language="C#">[System.Runtime.CompilerServices.Nullable(0)]</AttributeName>
+              <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.Nullable(0)&gt;]</AttributeName>
+            </Attribute>
+          </Attributes>
+          <Constraints>
+            <InterfaceName>Microsoft.Extensions.Http.Logging.IHttpClientLogger</InterfaceName>
+          </Constraints>
+        </TypeParameter>
+      </TypeParameters>
+      <Parameters>
+        <Parameter Name="builder" Type="Microsoft.Extensions.DependencyInjection.IHttpClientBuilder" Index="0" FrameworkAlternate="dotnet-plat-ext-8.0" RefType="this" />
+        <Parameter Name="wrapHandlersPipeline" Type="System.Boolean" Index="1" FrameworkAlternate="dotnet-plat-ext-8.0" />
+      </Parameters>
+      <Docs>
+        <typeparam name="TLogger">The service type of the custom logger as it was registered in DI. The logger should implement <see cref="T:Microsoft.Extensions.Http.Logging.IHttpClientLogger" /> or <see cref="T:Microsoft.Extensions.Http.Logging.IHttpClientAsyncLogger" />.</typeparam>
+        <param name="builder">The <see cref="T:Microsoft.Extensions.DependencyInjection.IHttpClientBuilder" />.</param>
+        <param name="wrapHandlersPipeline">Whether the logging handler with the custom logger would be added to the top or to the bottom of the additional handlers chains.</param>

```

Comment:
```suggestion
        <param name="wrapHandlersPipeline"><see langword="true" /> to add the logging handler with the custom logger to the top of the additional handlers chain; <see langword="false" /> to add it to the bottom of the chain.</param>
```

Reply:
Again, not sure if correct.

Reply:
@dotnet/area-extensions-dependencyinjection can you please confirm?

Reply:
Confirmed:
https://github.com/dotnet/runtime/blob/b41b63973bf49ad7a2d0d2ef37f08cf5df622ea1/src/libraries/Microsoft.Extensions.Http/src/DependencyInjection/HttpClientBuilderExtensions.Logging.cs#L58-L65

Reply:
@carlossanlop What do you think about the terms top and bottom? Do you think beginning and end are better? I do.

Reply:
I think the consumer would know what they refer to. We can always address these docs later.